geometry_msgs/PoseStamped pose_stamped
string frame
---
geometry_msgs/PoseStamped output_pose_stamped
